src/components/AddTodoForm.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+const AddTodoForm = () => {
+  return (
+    <form name="AddTodoForm">
+      <center>
+        <h1>Welcome to our Todo Form</h1>
+      </center>
+      <div className="todoForm">
+        <label htmlFor="TodoTitle">Todo Title</label>
+        <input type="text" className="input" id="TodoTitle" />
+        <label htmlFor="TodoName">Todo Name</label>
```

Comment:
You only should have one label & and input, todoTitle. That will accept the text of the todo. Also use camelCase, not PascalCase for `htmlFor` and `id`s.